DRAFT_RELEASE_NOTES.md

```diff
@@ -3,7 +3,7 @@
 ## TBD
 
 # Breaking Changes
-Note: instructions for adapting to these changes are outlined in the upgrade instructions below.
+We have upgraded the baseline to Java 17 and the minimal required Maven version is 3.9.6. There could be some deprecated classes or incompatible dependencies. Please update your dependency modules accordingly. 
```

Comment:
Rephrase: With the upgrade to Java 17, the new minimum required Maven version is now `3.9.6`. This can result in  deprecated classes or incompatible dependencies, please update your dependency modules accordingly.